src/query/service/src/schedulers/fragments/plan_fragment.rs

```diff
@@ -204,6 +206,8 @@ impl PlanFragment {
         let mut collect_read_source = |plan: &PhysicalPlan| {
             if let PhysicalPlan::TableScan(scan) = plan {
                 source.push(*scan.source.clone())
+            } else if let PhysicalPlan::DistributedCopyIntoTable(distributed_plan) = plan {
```

Comment:
2 if let -> match

Reply:
I think this is better?    if let PhysicalPlan::TableScan(scan) = plan {
            source.push(*scan.source.clone())
        }
        if let PhysicalPlan::DistributedCopyIntoTable(distributed_plan) = plan {
            source.push(*distributed_plan.source.clone())
        }
    use match  will cause the left branch meaningless?

Reply:
use match will cause the left branch meaningless? do you think that's better?

Reply:
otherwise give a unreachable()?